app/src/test/java/com/example/workoutapp/WorkoutUnitTest.kt

```diff
@@ -58,49 +53,34 @@ class WorkoutUnitTest {
-            "While holding the upper arms stationary, curl the weights forward while contracting the biceps as you breathe out. Tip: Only the forearms should move.",
-            "Continue the movement until your biceps are fully contracted and the bar is at shoulder level. Hold the contracted position for a second and squeeze the biceps hard.",
-            "Slowly begin to bring the bar back to starting position as your breathe in.",
-            "Repeat for the recommended amount of repetitions."))
+        assertTrue(workout.instructions.equals("Example Workout Instructions."))
         assertTrue(workout.category.equals("strength"))
     }
     //This test checks if the workout list is compiling the information correctly
     @Test
     fun `Given a workout dto when name is Barbell Curl force is pull etc`() {
-        var workout = Workout("Barbell Curl", "pull", "beginner", "isolation", "barbell", "biceps", "forearms",
-            "Stand up with your torso upright while holding a barbell at a shoulder-width grip. The palm of your hands should be facing forward and the elbows should be close to the torso. This will be your starting position.",
-            "While holding the upper arms stationary, curl the weights forward while contracting the biceps as you breathe out. Tip: Only the forearms should move.",
-            "Continue the movement until your biceps are fully contracted and the bar is at shoulder level. Hold the contracted position for a second and squeeze the biceps hard.",
-            "Slowly begin to bring the bar back to starting position as your breathe in.",
-            "Repeat for the recommended amount of repetitions.", "strength")
+        val workout = Workout("Barbell Curl", "pull", "beginner", "isolation", "barbell", "biceps", "forearms",
+            "Example Workout Instructions.", "strength")
         assertTrue(workout.ToString().equals("Barbell Curl pull beginner isolation barbell biceps forearms Stand up with your torso upright while holding a barbell at a shoulder-width grip. The palm of your hands should be facing forward and the elbows should be close to the torso. This will be your starting position. While holding the upper arms stationary, curl the weights forward while contracting the biceps as you breathe out. Tip: Only the forearms should move. Continue the movement until your biceps are fully contracted and the bar is at shoulder level. Hold the contracted position for a second and squeeze the biceps hard. Slowly begin to bring the bar back to starting position as your breathe in. Repeat for the recommended amount of repetitions. strength"))
     }
     //The json service we are using has split json files we need to parse so this will likely change as
     //we research how to make it work.
     @Test
-    fun `Given service connects to Countries JSON stream when data are read and parsed then country collection should be greater than zero`() =runTest {
```

Comment:
Good.  Function name should be descriptive.  "Countries" is a copy-paste artifact, and is misleading.